django_celery_results/managers.py

```diff
@@ -57,7 +57,44 @@ def _inner(*args, **kwargs):
     return _outer
 
 
-class TaskResultManager(models.Manager):
+class ResultManager(models.Manager):
```

Comment:
this is not a backward-incompatible change, right?

Reply:
right, I believe it is compatible. 

The git diff looks a little goofy, but what I've done is pulled out the common logic between task & group managers out into a parent class. `TaskResultManager` should act identically to how it does now in master

Reply:
make sense